LoopCore/CarbEntryUserInfo.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CarbEntryUserInfo.swift
+//  Naterade
+//
+//  Created by Nathan Racklyeft on 1/23/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+
+
+public struct CarbEntryUserInfo {
+    public let carbEntry: NewCarbEntry
+
+    public init(carbEntry: NewCarbEntry) {
+        self.carbEntry = carbEntry
+    }
+}
+
+
+extension CarbEntryUserInfo: RawRepresentable {
+    public typealias RawValue = [String: Any]
+
+    static let version = 2
+    public static let name = "CarbEntryUserInfo"
+
+    public init?(rawValue: RawValue) {
+        guard rawValue["v"] as? Int == type(of: self).version && rawValue["name"] as? String == CarbEntryUserInfo.name,
```

Comment:
This is used when sending a carb entry from the watch to the phone; since the watch app is usually updated when the app is, we shouldn't generally have a mismatch.